vignettes/adding-support-for-reporting.Rmd

```diff
@@ -31,19 +31,21 @@ The entire life cycle of objects involved in creating the report and configuring
 Let us consider an example module, based on the example module from `teal`:
 ```{r, message=FALSE}
 library(teal)
-teal_example_module <- function(label = "example teal module") {
+example_module <- function(label = "example teal module") {
   module(
     label,
     server = function(id, data) {
       moduleServer(id, function(input, output, session) {
+        ns <- session$ns
```

Comment:
```suggestion
        checkmate::assert_class(data, "reactive")
        checkmate::assert_class(data(), "teal_data")
        ns <- session$ns
```

Reply:
I decided to remove assertions from _this_ vignette because it concentrates on reporting and I wanted less clutter.
The examples in _Creating Custom Modules_ do have these assertions.

Reply:
Ah! please move them above moduleServer.

Reply:
Ah! please move them above moduleServer.

Reply:
Fair enough